index.html

```diff
@@ -2,9 +2,152 @@
+      <nav class="navbar py-0 navbar-light bg-dark justify-content-end">
+        <a class="nav-item nav-link" href="#"> <img src="images/angellist.png" alt="Tags"></a>
+        <a class="nav-item nav-link" href="#"> <img src="images/twitter.png" alt="Tags"> </a>
+        <a class="nav-item nav-link" href="#">English</a>
+        <a class="nav-item nav-link" href="#">My Page</a>
+        <a class="nav-item nav-link" href="#">Logout</a>
+      </nav>
+    </div>
+
+    <nav class="navig">
+      <img class="x" src="images/Enabled.png" alt="">
+      <div class="logo-con">
+        <a href="./index.html"><img class="logo" src="images/logo.png" alt=""></a>
+      </div>
+      <a href="./index-about.html">About</a>
+      <a href="#">Program</a>
+      <a href="#">Join</a>
+      <a href="#">Sponsor</a>
+      <a href="#">News</a>
+      <a href="#">Campains</a>
+    </nav>
+    <div class="wrapper1">
+      <img class="hamburger" src="images/hamburger1.png" alt="">
+      <div class="container p-5">
+        <div  class="container pt-0 p-3 mt-5 ">
+          <h4  id="h4">"Hello! Adding value to your journey"</h4>
+          <h1 id="h1">EPIC ENDGAME 24 DAYS  <br>SERIES PROGRAM <br> 2022</h1>
+        </div>
+        <div class="border border-white p-3 w-75" id="para">
+          <p>Let's really CRUSH this body workout together! <br>These workouts are going to challenge your body endurance and your strength.</p>
+        </div>
+       <div class="p-3">
+         <h4>2022.09.25(MON)~09.15(TU)</h4>
+         <p>@EPIC all rights reserved 2022.Epic game. 2020</p>
+       </div>
+      </div>
+    </div>
+
+    <div class="wrapper2">
+     <h5 class="text-prog">Main Program</h5>
+     <div class="line"></div>
+     <div class="card-wrap">
+       <div class="cart">
+         <img src="images/gym1.png" alt="">
+         <h5>Exercises Mixed</h5>
+         <p class="text-light" >Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
+       </div>
+       <div class="cart">
+         <img src="images/gym2.jpg" alt="">
+         <h5>Mixed Programs</h5>
+         <p class="text-light">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
+       </div>
+       <div class="cart">
+         <img src="images/gym3.png" alt="">
+         <h5>Weekly program</h5>
+         <p class="text-light">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
+       </div>
+       <div class="cart">
+         <img src="images/gym4.png" alt="">
+         <h5>Months program</h5>
+         <p class="text-light">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
+       </div>
+       <div class="cart">
+         <img src="images/gym5.jpg" alt="">
+         <h5>Yearly programs</h5>
+         <p class="text-light">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
+       </div>
+     </div>
+     <div class="btn-container">
+       <button type="button" class="button">Join The EpicGame Program 2022</button>
+     </div>
+    </div>
```

Comment:
- [ ] According to the requirements and the original design, You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including Colors, Typography, and Layout. So kindly, on the desktop version, you should update/change the style of the `SEE THE WHOLE PROGRAM` Link, increase the width of the icons, and increase the card padding (top and bottom) to look similar to the original design 💯👌

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/181512971-12b23168-48ef-4179-8f18-40836cc18fbd.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/181513222-2cb7a487-6b4d-46a8-b006-3b92c32beb92.png" width="500" /> |